contracts/Feature.sol

```diff
@@ -487,7 +488,6 @@ contract Feature is Initializable, NativeMetaTransaction, ChainConstants, Contex
         uint256 delayClaim; // Time of the challenge period.
         string metaEvidence; // Link to the meta-evidence.
         uint256 runningClaimCount; // Count of running claims.
-        bool isExecuted;
     }
```

Comment:
```
   struct Transaction {
        bool isExecuted; // 1 bytes
        address sender; // 20 bytes
        Arbitrator arbitrator; // 20 bytes
        bytes arbitratorExtraData; // dynamic
        uint256 delayClaim; // 32 bytes
        string metaEvidence; // dynamic
        uint256 runningClaimCount; // 32 bytes
    }
    ```
    
    So I don't think there is any optimization in this case.
    
    BTW it's more conveniant to have these variables orders to understand better the struct.